test/IDE/range_info_basics.swift

```diff
@@ -144,6 +144,11 @@ func testExpressionVariables() -> Int {
   return bar()
 }
 
+func testDefer() {
+  defer {
+  }
+}
```

Comment:
@rintaro does this test fail without your changes? I thought I'd tried this one when I was reducing the original test case with no luck, but might be misremembering.